package-lock.json

```diff
@@ -6995,7 +6995,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
Please revert the changes in this file. We don't need to be changing anything in the packagelock.json file.
Thanks!